src/components/SubNav/SubNav.tsx

```diff
@@ -0,0 +1,272 @@
+import React, { forwardRef } from "react";
+import {
+  Box,
+  chakra,
+  ChakraComponent,
+  useMultiStyleConfig,
+  Flex,
+} from "@chakra-ui/react";
+import Button from "../Button/Button";
+import Link from "../Link/Link";
+import List from "../List/List";
+import useScrollFadeStyles from "../../hooks/useScrollFadeStyles";
+
+export const actionBackgroundColorsArray = [
```

Comment:
Both arrays of colors are already declared in `Banner`– could they live in one place now that two components are using them?